### DISCUSSION TITLE
[question] how would you approach motion blur?

### DISCUSSION THREAD

**Discussion by zhuker** [nvpro-samples/vk_raytracing_tutorial_KHR#14] [General]
i have a couple timed instance transforms and a bunch of ray times (90k distinct times) 
my first naive implementation updates TLAS and retraces rays for each time but it obviously takes forever (90k retraces per frame)
how would you do it?

an informal request to extend the tutorial

**Comment by mklefrancois** [comment]
Sorry for the late answer but there is now an extension for motion bur. Please have a look at https://github.com/nvpro-samples/vk_raytracing_tutorial_KHR/tree/master/ray_tracing_motionblur
